lib/features/tbdex/quote_notifier.dart

```diff
@@ -31,7 +31,7 @@ class QuoteAsyncNotifier extends AutoDisposeAsyncNotifier<Quote?> {
   @override
   FutureOr<Quote?> build() => null;
 
-  void startPolling(String exchangeId, Pfi pfi) {
+  void startPolling(String? exchangeId, Pfi? pfi) {
```

Comment:
Should we just return here if there's no exchangeId or pfi? What would happen if we keep calling `getExchange` with these being null?